src/exec/job.rs

```diff
@@ -25,8 +25,8 @@ pub fn job(
 
         // Obtain the next result from the receiver, else if the channel
         // has closed, exit from the loop
-        let value: PathBuf = match lock.recv() {
-            Ok(WorkerResult::Entry(path)) => path,
+        let value: DirEntry = match lock.recv() {
+            Ok(WorkerResult::Entry(val)) => val,
```

Comment:
In the meantime, this has been changed on `master`. Could we change this here accordingly (`path` instead of `val`)?

Reply:
well no. Actually now it's a `dir_entry`. So let's change `val` to `dir_entry`. And also `value` => `dir_entry`.